udata-tabular-preview/locales/fr.json

```diff
@@ -1,12 +1,13 @@
 {
   "Preview of {name}": "Prévisualisation de {name}",
   "{count} columns": "{count} colonnes | {count} colonne | {count} colonnes",
-  "Explore data": "Explorer les données",
+  "Explore data": "Explorer",
   "The preview of this file failed to load.": "La prévisualisation de ce fichier n'a pas pu être chargée.",
   "{count} rows": "{count} lignes | {count} ligne | {count} lignes",
   "Sort ascending": "Trier par ordre croissant",
   "Sort descending": "Trier par ordre décroissant",
   "The data structure of this file failed to load.": "La structure de données de ce fichier n'a pas pu être chargée.",
-  "This is an Excel file, analysis on this type of files are limited.": "Il s'agit d'un fichier Excel, l'analyse sur ce type de fichiers est limitée.",
-  "No data structure found for this file.": "Aucune structure de données trouvée pour ce fichier."
+  "No data structure found for this file.": "Aucune structure de données trouvée pour ce fichier.",
+  "Explore data in detail": "Explorer les données en détail",
+  "Use our tool": "Utiliser notre outil pour obtenir un aperçu des données, en savoir plus sur les différentes colonnes ou réaliser des filtres et des tris."
```

Comment:
We usually do these translations in Crowdin. I'm okay applying these directly if it doesn't raise issue. What do you think @nicolaskempf57?